embulk-docs/src/built-in.rst

```diff
@@ -128,17 +128,19 @@ The ``file`` input plugin reads files from local file system.
 Options
 ~~~~~~~~
 
-+----------------+----------+------------------------------------------------+-----------+
-| name           | type     | description                                    | required? |
-+================+==========+================================================+===========+
-| path\_prefix   | string   | Path prefix of input files                     | required  |
-+----------------+----------+------------------------------------------------+-----------+
-| parsers        | hash     | Parsers configurations (see below)             | required  |
-+----------------+----------+------------------------------------------------+-----------+
-| decoders       | array    | Decoder configuration (see below)              |           |
-+----------------+----------+------------------------------------------------+-----------+
-| last\_path     | string   | Name of last read file in previous operation   |           |
-+----------------+----------+------------------------------------------------+-----------+
++-----------------+----------+------------------------------------------------+----------------------+
+| name            | type     | description                                    | required?            |
++=================+==========+================================================+======================+
+| path\_prefix    | string   | Path prefix of input files                     | required             |
++-----------------+----------+------------------------------------------------+----------------------+
+| parsers         | hash     | Parsers configurations (see below)             | required             |
++-----------------+----------+------------------------------------------------+----------------------+
+| decoders        | array    | Decoder configuration (see below)              |                      |
++-----------------+----------+------------------------------------------------+----------------------+
+| last\_path      | string   | Name of last read file in previous operation   |                      |
++-----------------+----------+------------------------------------------------+----------------------+
+| follow_symlinks | boolean  | Follow symbolic link directories               | ``false`` by default |
```

Comment:
Is this following only symbolic link "directories"? The definition of `FOLLOW_LINKS` does not say so.
http://docs.oracle.com/javase/jp/7/api/java/nio/file/FileVisitOption.html#FOLLOW_LINKS

I'd suggest `If true, follow symbolic links` for this.